### ISSUE TITLE
Warnings about cannot import

### ISSUE THREAD

**Issue by musm** [JuliaStrings/LegacyStrings.jl#35]
when running on julia 1.0
```julia
WARNING: could not import Base.lcfirst into LegacyStrings
WARNING: could not import Base.next into LegacyStrings
WARNING: could not import Base.rsearch into LegacyStrings
WARNING: could not import Base.search into LegacyStrings
WARNING: could not import Base.ucfirst into LegacyStrings
```

**Comment by stevengj** [comment]
This should be fixed to use the new names… `uppercasefirst`, `findlast`, `iterate`, etcetera.

**Comment by stevengj** [comment]
Closed by #36
